### PR TITLE
Minor Updates to the PackageDescription In-Source Documentation

### PR DESCRIPTION
**Pull request by ElektrojungeAtWork**
This PR includes some minor updates to the in-source documentation to address editorial feedback.

### CODE REVIEW

File: Sources/PackageDescription/Target.swift [hunk 1]
```diff
@@ -275,24 +273,22 @@ public final class Target {
...
     /// - Parameters:
     ///   - name: The name of the target.
     ///   - dependencies: The dependencies of the target. A dependency can be another target in the package or a product from a package dependency.
-    ///   - path: The custom path for the target. By default, the Swift Package Manager expects a target's sources to reside at predefined search paths,
+    ///   - path: The custom path for the target. By default, the Swift Package Manager requires a target's sources to reside at predefined search paths;
     ///       for example, `[PackageRoot]/Sources/[TargetName]`.
-    ///       Don't escape the package root; that is, values like `../Foo` or `/Foo` are invalid.
+    ///       Don't escape the package root; for example, values like `../Foo` or `/Foo` are invalid.
     ///   - exclude: A list of paths to files or directories that the Swift Package Manager shouldn't consider to be source or resource files.
-    ///       This path is relative to the target's directory.
```
Comment (neonichu) [inline Sources/PackageDescription/Target.swift:291]: This makes it sound kind of unrelated to the former, doesn't it? How about "Each of these paths" or something like that?

Comment (ElektrojungeAtWork) [reply Sources/PackageDescription/Target.swift:291]: How about `Paths are relative to the target's directory.`?

Comment (abertelrud) [reply Sources/PackageDescription/Target.swift:291]: That wording sounds good.

Comment (ElektrojungeAtWork) [reply Sources/PackageDescription/Target.swift:291]: Done.

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Comment by ElektrojungeAtWork** [comment]
@swift-ci please smoke test

**Comment by neonichu** [comment]
@swift-ci please smoke test

**Pull request merged**